webpack.config.js

```diff
@@ -28,6 +28,6 @@ module.exports = {
     },
     disableHostCheck: true
   },
-  externals: ["single-spa"],
+  externals: ["single-spa", "i18next", "react-i18next"],
```

Comment:
This commits us to putting i18next and react-i18next into the import map. I have already done so for openmrs-spa.org (see https://openmrs-spa.org/openmrs/frontend/import-map.json)